src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3028,9 +3075,13 @@ private void BuildStyleProperty(IIndentedStringBuilder writer, XamlObjectDefinit
 
 		private bool IsLocalizedString(INamedTypeSymbol propertyType, string objectUid)
 		{
-			return objectUid.HasValue()
-				&& (propertyType.ToDisplayString() == "string"
-				|| propertyType.ToDisplayString() == "object");
+			return objectUid.HasValue() && IsLocalizablePropertyType(propertyType);
+		}
+
+		private static bool IsLocalizablePropertyType(INamedTypeSymbol propertyType)
+		{
+			return propertyType?.ToDisplayString() == "string"
```

Comment:
Try using _stringSymbol and _objectSymbol. Those will get used a lot, and it will avoid a string generation.

Reply:
fixed :)